source/kernel/src/comms/kchannel.rs

```diff
@@ -55,11 +55,34 @@ impl<T> Deref for KChannel<T> {
     }
 }
 
+fn right_size(mut cap: usize) -> usize {
+    if cap < 2 {
+        tracing::warn!(
```

Comment:
CI fails because we need a `use crate::tracing;`: https://github.com/tosc-rs/mnemos/actions/runs/5650604492/job/15307247587?pr=185#step:8:477